src/status_im/chat/new_public_chat/view.cljs

```diff
@@ -2,11 +2,11 @@
   (:require-macros [status-im.utils.views :refer [defview letsubs]])
   (:require [re-frame.core :refer [dispatch]]
             status-im.utils.db
-            [status-im.components.react :refer [view text]]
+            [status-im.components.react :as react :refer [text]]
```

Comment:
why did you keep refer for text?

Reply:
I missed it somehow. I didn't migrate everything because they were too many and wanted to keep it focused.